src/Proto.Cluster/Identity/IdentityStorageWorker.cs

```diff
@@ -47,9 +47,18 @@ public Task ReceiveAsync(IContext context)
                 return Task.CompletedTask;
             }
 
-            if (_cluster.PidCache.TryGet(msg.ClusterIdentity, out var existing))
+            var clusterIdentity = msg.ClusterIdentity;
+        //    var ct = msg.CancellationToken;
+            //
+            // if (ct.IsCancellationRequested)
+            // {
+            //     _logger.LogError("CT already timed out....");
+            //     return Task.CompletedTask;
+            // }
```

Comment:
if IdentityWorker receives a message, where the CT already have timed out, what should we do with that?
discard? any future waiting for a response is already dead.

Reply:
Seems reasonable to discard it then, perhaps return null to sender. We could also link the global cancellationtoken with these.

Reply:
And if we should log error, we should probably throttle it :)